### ISSUE TITLE
Can't use string as Paragraph children

### ISSUE THREAD

**Issue by la55u** [tamagui/tamagui#984]
### Current Behavior

When a variant is set on a `styled(Paragraph)` it cannot be used as a regular Paragraph, in other words it does not accept string as children.

The error is:
```
Type '{ children: string; open: true; }' is not assignable to type 'Omit<{ readonly size?: FontSizeTokens | undefined; } & { [x: string]: undefined; }, "open">'.
  Property 'children' is incompatible with index signature.
    Type 'string' is not assignable to type 'undefined'.
```

### Expected Behavior

I should be able to use styled Paragraps as `<MyParagraph open>Something</MyParagraph>`

### Tamagui Version

```markdown
1.5.17
```

### Reproduction

Repo (see the commit history):
https://github.com/la55u/tama-paragraph-bug

### System Info

```markdown
System:
    OS: Linux 6.2 Fedora Linux 37 (Workstation Edition)
    CPU: (16) x64 12th Gen Intel(R) Core(TM) i7-1260P
    Memory: 21.12 GB / 31.04 GB
    Container: Yes
    Shell: 5.2.15 - /bin/bash
  Binaries:
    Node: 18.12.1 - ~/.nvm/versions/node/v18.12.1/bin/node
    Yarn: 3.5.0 - ~/.nvm/versions/node/v18.12.1/bin/yarn
    npm: 8.19.2 - ~/.nvm/versions/node/v18.12.1/bin/npm
  Browsers:
    Chrome: 112.0.5615.121
    Firefox: 113.0b4
  npmPackages:
    @babel/runtime: ^7.18.9 => 7.20.13 
    @manypkg/cli: ^0.19.1 => 0.19.2 
    eslint: ^8.21.0 => 8.32.0 
    prettier: ^2.7.1 => 2.8.3 
    turbo: ^1.8.3 => 1.8.3 
    typescript: ^4.7.4 => 4.9.4
```

**Comment by deggertsen** [comment]
Thanks for creating a reproduction. This issue is preventing me from upgrading to recent versions.

**Comment by natew** [comment]
Fixed in 9483863370059355bf55bce6fba79a165c0a1f94 patch release going out now
